js/logic.js

```diff
@@ -22,11 +22,11 @@ var semesterManager = {
 		"SS17"
 	],
 	numberDisplayed: 6,
-	current: "SS15",
-	lastSummerSemester: "SS14",
+	current: "WS15/16",
+	lastSummerSemester: "SS15",
 	lastWinterSemester: "WS14/15",
```

Comment:
Please change `lastWinterSemester` to `"WS15/16"`. That was wrong before already, but `current` should always be one of `lastSummerSemester` and `lastWinterSemester`.
I should add a comment there.